### DISCUSSION TITLE
画了个简捏

### DISCUSSION THREAD

**Discussion by yuyu5201314** [share121/inter-knot#3355] [其他]
这张图我在pixiv和推都发过，不过没啥人看……来这里碰碰运气，其他平台id是予雨
![Uploading mmexport1724007639172.png…]()

**Comment by sicusa** [comment]
图片没有上传成功诶，要注意github里上传图片需要等“Uploading...”这行文本消失，然后变成实际的图片才能点 Comment

**Reply by yuyu5201314** [reply]
原来如此嘛🥁

**Comment by yuyu5201314** [comment]
![mmexport1724007949723](https://github.com/user-attachments/assets/67d8a3d8-e749-4c26-bf3c-aab00b223f30)
帖子里图片没传上去，这里补一下🥺
